build_tools/kokoro/gcp_ubuntu/cmake-bazel/linux/x86-swiftshader/build.sh

```diff
@@ -57,13 +57,16 @@ echo "Configuring CMake"
    -DCMAKE_BUILD_TYPE=RelWithDebInfo \
    -DIREE_BUILD_COMPILER=ON \
    -DIREE_BUILD_TESTS=ON \
+   -DIREE_BUILD_BENCHMARKS=ON \
    -DIREE_BUILD_SAMPLES=OFF \
    -DIREE_BUILD_XLA_COMPILER=ON \
    -DIREE_BUILD_TFLITE_COMPILER=ON \
    -DIREE_BUILD_TENSORFLOW_COMPILER=ON .
 
 echo "Building with Ninja"
 cd "${CMAKE_BUILD_DIR?}"
+
+ninja iree-benchmark-suites
 ninja
```

Comment:
You can list multiple targets

```suggestion
ninja all iree-benchmark-suites
```

Reply:
It's for checking the dependency is properly set in CMake. If just `ninja` first, all dependencies will be built. That can hide issues. (Like I run into them and fixed with https://github.com/google/iree/pull/5907.)

Reply:
I'm fine to swap or add a comment to be clear.